collector/benchmarks/cranelift-codegen-0.82.1/perf-config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "category": "primary"
+}
```

Comment:
The `perf-config.json` for the old `cranelift-codegen` has a `cargo_toml` and a `touch_file` entry. Are they needed here too?

Reply:
I didn't expect them to be: the old benchmark is a workspace with in-tree members, i.e. likely a git clone of the entire cranelift repo, and I believed it used these settings to only profile `cranelift-codegen` amongst all the sub-crates. The new benchmark is just the crate itself from crates.io, and is not a workspace: the default behavior of profiling a library should work fine ?

Reply:
Makes sense, thanks!

Reply:
I will launch other runs of the benchmark, with a more realistic workload, to make sure